conf/defaults.ini

```diff
@@ -420,6 +420,12 @@ user_invite_max_lifetime_duration = 24h
 # Enter a comma-separated list of usernames to hide them in the Grafana UI. These users are shown to Grafana admins and to themselves.
 hidden_users =
 
+[service_accounts]
+# Service account maximum expiration date in days.
+# Warning: When set, Grafana reduces the lifespan of _all existing tokens_ that have a days-to-live value greater than `token_expiration_day_limit` in order to comply with this setting.
+# This action is irreversible; you can't extend existing tokens' days-to-live by changing this setting to a larger value.
+token_expiration_day_limit = -1
```

Comment:
have you considered making this option not set by default?

otherwise it's a setting that has been set to some value.
the idea is that we do not set a value until it is needed

Let me know your thoughts and if this makes it so that you have to change alot.
This might be necessary for the frontend to work?

Reply:
I'm not strongly opinionated on this. My reasoning was to leave `-1` as a quick example of a value.

If you reply with a 🚀 to this comment, I'll can change it for default.ini and sample.ini

Reply:
we had a quick chat and said that we would take a look at some of the code changes that's needed and review if this is worth or too nit picky 💅